### PR TITLE
Add sleep time for each file upload try

### PR DESCRIPTION
**Pull request by jdneo**
When upload file (*.jar) to web app, sometime it may occur ftp 550 error. The reason may bethe *jar file not been released. So we add upload retry and for each try, the thread will sleep 5s first.

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppRunState.java [hunk 1]
```diff
@@ -86,6 +86,7 @@
     private static final String ROOT_PATH = BASE_PATH + "ROOT";
     private static final String CONTAINER_ROOT_PATH = WEB_APP_BASE_PATH + "ROOT";
 
+    private static final int SLEEP_TIME = 5000;
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppRunState.java]: add comments to show the unit.

**Comment by jdneo** [comment]
retest this please.

**Comment by jdneo** [comment]
retest this please

**Comment by jdneo** [comment]
retest this please.

**Pull request merged**